mcux/mcux-sdk/components/css_pkc/src/platforms/LPC55S3x/inc/ip_platform.h

```diff
@@ -20,7 +20,7 @@
 #ifndef IP_PLATFORM_H
 #define IP_PLATFORM_H
 
-#include "board.h"
```

Comment:
Changes to the SDK should be sent to the SDK github repo ((https://github.com/NXPmicro/mcux-sdk) as a PR. This way we don't have to maintain local SDK changes inside Zephyr.

Reply:
PR submitted

Reply:
Please also update the PATCH section of the README file with details about the files changes so we don't lose these changes on an SDK Update